.github/workflows/ci.yml

```diff
@@ -12,49 +12,49 @@ concurrency:
 
 jobs:
   lint:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
Note: `ubuntu-latest` used to serve an old version in GitHub Actions, but now is current (same for `macos`).